main/pipeline/pipeline.cc

```diff
@@ -1264,4 +1264,43 @@ vector<ast::ParsedFile> autogenWriteCacheFile(const core::GlobalState &gs, const
 #endif
 }
 
+void printUntypedBlames(const core::GlobalState &gs, const UnorderedMap<long, long> untypedBlames,
+                        const std::string &blameFilePath) {
+#ifndef SORBET_REALMAIN_MIN
+
+    rapidjson::StringBuffer result;
+    rapidjson::Writer<rapidjson::StringBuffer> writer(result);
+
+    writer.StartArray();
+
+    for (auto e : untypedBlames) {
+        auto sym = core::SymbolRef::fromRaw(e.first);
+        if (!sym.exists() || !sym.loc(gs).exists()) {
+            continue;
+        }
+
+        writer.StartObject();
+
+        writer.String("path");
+        writer.String(std::string(sym.loc(gs).file().data(gs).path()));
+
+        writer.String("owner");
+        auto owner = sym.owner(gs).show(gs);
+        writer.String(std::string(owner));
```

Comment:
iiuc, `owner` is already a string, right?
```suggestion
        writer.String(owner);
```